### PR TITLE
Mk 7 apply google formatter

### PR DESCRIPTION
**Pull request by michalkozminski**
Use google code style for linting

### CODE REVIEW

File: beekeeper-formatter-plugin/src/main/java/io/beekeeper/formatter/FormatterPlugin.java [hunk 1]
```diff
@@ -54,10 +47,9 @@ private void applySpotlessJavaConfiguration(Project project) {
         // our plugin before the java plugin
         project.getPluginManager().withPlugin(PLUGIN_JAVA, javaPlugin -> {
             extension.java(java -> {
+                java.googleJavaFormat();
```
Comment (isacandrei) [inline beekeeper-formatter-plugin/src/main/java/io/beekeeper/formatter/FormatterPlugin.java:50]: would be very nice we we add the import order that we'd like here as well.

File: gradle.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
 org.gradle.caching=true
-version=0.11.1
+version=0.12.1
```
Comment (isacandrei) [inline gradle.properties:2]: Do we have anywhere a changelog? If not, maybe we should start one in the main wiki.

Comment (michalkozminski) [reply gradle.properties:2]: Didn't see any are you aware of one?

File: beekeeper-formatter-plugin/src/main/java/io/beekeeper/formatter/FormatterPlugin.java (hunk 1 above)
Comment (michalkozminski) [reply beekeeper-formatter-plugin/src/main/java/io/beekeeper/formatter/FormatterPlugin.java:50]: would it matter with the formatter?

Comment (isacandrei) [reply beekeeper-formatter-plugin/src/main/java/io/beekeeper/formatter/FormatterPlugin.java:50]: hmm, they do have their own ordering definition. I guess we can go with that.

File: gradle.properties (hunk 2 above)
Comment (isacandrei) [reply gradle.properties:2]: Not really, but maybe we can start one in the main readme file.

Comment (michalkozminski) [reply gradle.properties:2]: 👍

**Comment by michalkozminski** [comment]
@isacandrei shall we merge it?

**Review by isacandrei — APPROVED** [review]
LGTM

**Pull request merged**